File: Domain/Candidate.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace Domain
{
    public class Candidate
    {

        public Guid Id { get; set; }
        public string Email { get; set; }

        public string FirstName { get; set; }

        public string Prefix { get; set; }

        public string Lastname { get; set; }

        public bool? Searching { get; set; }

        public int? Rate { get; set; }

        public int? Hours { get; set; }

        public string Role { get; set; }

        public DateTime? Availability { get; set; }

        public string PhoneNumber { get; set; }

        public string City { get; set; }

        public string FileName { get; set; }

        public bool? AgreedTermsAndConditions { get; set; }

        public DateTime? AgreedTermsAndConditionsDate { get; set; }

        public int? KvkNummer { get; set; }

        public int? AssignmentSearchRadius { get; set; }

        public Guid DiscountGuid { get; set; }

        public bool? Whatsapp { get; set; }

        public string DefaultMotivation { get; set; }
        
    }
}

File: client-app/src/app/models/candidate.ts

export interface Candidate {
    id: string;
    email: string;
    firstName: string;
    prefix: string;
    lastname: string;
    searching?: boolean;
    rate?: number;
    hours?: number;
    role: string;
    availability?: Date;
    phoneNumber: string;
    city: string;
    fileName: string;
    agreedTermsAndConditions?: boolean;
    areedTermsAndConditionsDate?: Date;
    kvkNummer?: number;
    assignmentSearchRadius?: number;
    discountGuid: string;
    whatsapp?: boolean;
    defaultMotivation: string;
  }